frontend/scss/components/templates/roadmap.scss

```diff
@@ -281,18 +306,24 @@
       }
 
       &-issue {
+        position: relative;
         min-height: 200px;
-        padding: 15px 15px 5px 20px;
+        padding: 15px 15px 8px 20px;
         border-radius: 8px;
         border-left: 7px solid color('black');
         background-color: color('white');
-        box-shadow: 0 2px 25px 0 rgba(0, 0, 0, 0.07);
-        transition: box-shadow .3s ease-in;
+        box-shadow: 0 2px 18px 0 rgba(0, 0, 0, 0.07);
```

Comment:
It'd be nice if all those raw `rgba` statements would use  `transparentize`/`fade-out` and our color variables instead:

```rgba
box-shadow: 0 2px 18px 0 fade-out(color('black'), 0.93);
```